frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/SearchEstimate/MobileSearchApprovedEstimates.js

```diff
@@ -0,0 +1,133 @@
+import React, { Fragment, useCallback, useMemo, useReducer } from "react";
+import { Link } from "react-router-dom";
+import { CloseSvg, SearchForm, Table, Card, SearchAction, PopUp, DetailsCard, Loader, Toast, CreateLoiIcon } from "@egovernments/digit-ui-react-components";
+import SearchFormFields from "./SearchFields";
+
```

Comment:
@burhanj-eGov Search approved estimate mobile view is already merged. I am assuming this file is for search estimates, pls change the file name

Reply:
DOne